scipy/spatial/_spherical_voronoi.py

```diff
@@ -16,9 +16,21 @@
 import scipy
 import itertools
 from . import _voronoi
+from scipy.spatial.distance import pdist
 
 __all__ = ['SphericalVoronoi']
 
+def sphere_check(points, radius, center):
+    """ Determines distance of generators from theoretical sphere
+    surface.
+
+    """
+    actual_squared_radii = (((points[...,0] - center[0]) ** 2) +
+                            ((points[...,1] - center[1]) ** 2) +
+                            ((points[...,2] - center[2]) ** 2))
+    max_discrepancy = (actual_squared_radii - (radius ** 2)).max()
```

Comment:
This would be better to write as `sqrt(actual_squared_radii) - radius`.
Below, it is compared to `threshold` which has the unit of euclidean length (in the pdist check), not length squared.